datalad_debian/add_distribution.py

```diff
@@ -0,0 +1,80 @@
+import logging
+
+from datalad.distribution.dataset import (
+    EnsureDataset,
+    datasetmethod,
+    require_dataset,
+)
+from datalad.interface.base import (
+    Interface,
+    build_doc,
+)
+from datalad.interface.utils import (
+    eval_results,
+)
+from datalad.support.constraints import (
+    EnsureNone,
+    EnsureStr,
+)
+from datalad.support.param import Parameter
+
+lgr = logging.getLogger('datalad.debian.add_distribution')
+
+
+@build_doc
+class AddDistribution(Interface):
+    """Create a distribution dataset to an package repository dataset
```

Comment:
Nit: "an" -> "a"

I bump a bit on the phrase "package repository dataset".

I suggest something like (which more closely follows the terms @loj defined in the docs):

> Create a distribution dataset in a Debian archive repository dataset